app/screens/thread/thread.tsx

```diff
@@ -58,20 +57,7 @@ const Thread = ({
     }, [componentId]);
 
     useAndroidHardwareBackHandler(componentId, close);
-
-    useEffect(() => {
-        const listener = {
-            componentDidAppear: () => {
-                RNUtils.setSoftKeyboardToAdjustNothing();
-            },
-            componentDidDisappear: () => {
-                RNUtils.setSoftKeyboardToAdjustResize();
-            },
-        };
-        const unsubscribe = Navigation.events().registerComponentListener(listener, componentId!);
-
-        return () => unsubscribe.remove();
-    }, []);
+    useAndroidAdjustSoftKeyboard(componentId);
```

Comment:
Do we need to consider other screens where the keyboard appears, or is it only for those where we are moving the input box around?

Reply:
Only the ones that apply extra keyboard really